eng/scripts/Publish-MavenPackages.ps1

```diff
@@ -266,18 +280,21 @@ foreach ($packageDetail in $packageDetails) {
 
   if ($RepositoryUrl -match "https://pkgs.dev.azure.com/azure-sdk/\b(internal|public)\b/*") {
     Write-Information "GPG Signing and deploying package in one step to devops feed: $RepositoryUrl"
+    Write-Information "mvn gpg:sign-and-deploy-file `"--batch-mode`" `"$pomOption`" `"$fileOption`" `"$javadocOption`" `"$sourcesOption`" `"$filesOption`" $classifiersOption `"$typesOption`" `"-Durl=$RepositoryUrl`" `"$gpgexeOption`" `"-DrepositoryId=target-repo`" `"-Drepo.password=$RepositoryPassword`" `"--settings=$PSScriptRoot\..\maven.publish.settings.xml`""
```

Comment:
Should this write the password?

Reply:
No, anything that's a secret gets *'d out. This is the actual output in the step
`-RepositoryPassword ***`